<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>470</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Helvetica</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="5">
      <widget class="QPushButton" name="pushButton_6">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_ACOS">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>acos</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QPushButton" name="pushButton_MUL">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_SIN">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>sin</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QPushButton" name="pushButton_MINUS">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="pushButton_3">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_LN">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>ln</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="pushButton_DIVIDE">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>รท</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QPushButton" name="pushButton_LEFT_SKOBKA">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="QPushButton" name="pushButton_RIGHT_SKOBKA">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_POW">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>^</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushButton_8">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pushButton_1">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="pushButton_5">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_ATAN">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>atan</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_COS">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>cos</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_LOG">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>log</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pushButton_MOD">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>mod</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_SQRT">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>sqrt</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="pushButton_2">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="pushButton_9">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_4">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_TAN">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>tan</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="pushButton_DELETE">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>AC</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QPushButton" name="pushButton_EQUAL">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="pushButton_0">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QPushButton" name="pushButton_PLUS">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_7">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_ASIN">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>asin</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QPushButton" name="pushButton_TOCHKA">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="pushButton_X">
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(255, 105, 114);
color: rgb(255, 146, 229);
</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="7">
      <widget class="QLabel" name="result_show">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>23</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
